mobile_verifier/src/heartbeats/mod.rs

```diff
@@ -538,17 +537,35 @@ impl ValidatedHeartbeat {
                         true
                     }
                 };
+
                 let distance_to_asserted = asserted_latlng.distance_m(hb_latlng).round() as i64;
-                let location_trust_score_multiplier = if is_valid
-		// The heartbeat location to asserted location must be less than the max_distance_to_asserted value:
-                    && distance_to_asserted <= max_distance_to_asserted as i64
-		// The heartbeat location to every associated coverage hex must be less than max_distance_to_coverage:
-		    && coverage_object.max_distance_m(hb_latlng).round() as u32 <= max_distance_to_coverage
-                {
-                    dec!(1.0)
+                let max_distance = coverage_object.max_distance_m(hb_latlng).round() as u32;
+
+                let location_trust_score_multiplier = if !is_valid {
+                    // QUESTION: Invalid heartbeats dropped the location trust multiplier to 0.25x.
```

Comment:
remember to remove these questions given the answers have been confirmed

Reply:
Removed, thank you